tslib/src/caching/redis-service.ts

```diff
@@ -40,27 +49,100 @@ export class RedisService implements IRedisService {
         host: config.Redis.Host
       }
     }
-    this._redis = new Redis(options)
+    this.redis = new Redis(options)
+   /* this.redis
```

Comment:
non blocker about removing the comments unless they server a reference purpose.

Reply:
Apologies George - i merged it in a few moments ago, before this comment landed.  I only retained those event binders for debugging, so will remove them in my next PR. 

thanks

Reply:
alright thanks